go-controller/vendor/github.com/containernetworking/plugins/pkg/ip/cidr.go

```diff
@@ -19,43 +19,87 @@ import (
 	"net"
 )
 
-// NextIP returns IP incremented by 1
+// NextIP returns IP incremented by 1, if IP is invalid, return nil
```

Comment:
According to this the function can return nil if the IP is invalid. Now that you migrated to this function should we be checking the return value of this function to see if it is nil?

Reply:
it is obvious the passed IP would be valid for most of callers. I will fix the others, especially for the utility functions that calling this function.

Reply:
I think you need to fix this in go-controller/pkg/ovn/base_secondary_layer2_network_controller.go and go-controller/hybrid-overlay/pkg/controller/node_windows.go as well which are still using the vendored functions directly?